### ISSUE TITLE
Bootstrap indexing issue/question

### ISSUE THREAD

**Issue by XhmikosR** [algolia/docsearch-configs#1137]
# Do you want to request a **feature** or report a **bug**?

### If it is a DocSearch index issue, what is the related `index_name` ?

`index_name`= `bootstrap` or `bootstrap-v4`?

## What is the **current** behaviour?

v4.4.0 isn't indexed

Hello!

Yesterday we released v4.4.0. I was under the impression that this would autoupdate, since we are using these meta tags:

```html
<meta name="docsearch:language" content="en">
<meta name="docsearch:version" content="4.4">
```

Maybe I misunderstood how this was going to work, so can you please shed some light on this so that we don't hit it again?

Thanks!

**Comment by s-pace** [comment]
👋 @XhmikosR 

Congrats for the release!

The DocSearch meta tags are missing on the pages `4.0`, `4.1` & `4.2`. This is mitigated by this [change of your configuration](https://github.com/algolia/docsearch-configs/commit/cc62a5c11da81339137d9c744d714d1d632e9501). It used to be mitigated by the `(?P<version>.*?)` variables.

The index `bootstrap` is used on 4.1 and following pages while the index `bootstrap-v4` is only used on the 4.0 pages: https://getbootstrap.com/docs/4.0/.

The `bootstrap` index is automatically updated with the new release since it relies on the meta tags that are present on 4.3 and following. You can see that the search is working on the 4.4 pages without updating your configuration. This is why I have closed #1138. 

Can you make sure to add these meta to the 4.0, 4.1 and 4.2 pages please? Once these metas are available from your DOM and that you have updated the 4.0 pages with the following credentials, we can safely remove `bootstrap-v4` and only rely on `bootstrap`. This will be done by merging #1140.

  apiKey: '5990ad008512000bba2cf951ccf0332f',
  indexName: 'bootstrap',

Please note that you can already use `bootstrap` on the 4.0 pages.

Bonus: The cells information were not retrievable from the search, it is now fixed [thanks to this update of your configuration](https://github.com/algolia/docsearch-configs/commit/f735b08a0ee6d42b1426c3aad4a42cef13956c16). You can try it out.

Let us know if you need any more details.

**Comment by s-pace** [comment]
👋 @XhmikosR

Any updates on this topic? Do you need more assistance?

**Comment by XhmikosR** [comment]
Hi, @s-pace and sorry for the late reply; I missed the email.

We don't plan to change how the old docs are. Which means the old versions need to stay as is. We added the metatags for 4.3+ so that the whole thing is completely automated but as you saw, it isn't.

**Comment by s-pace** [comment]
Hi @XhmikosR 

Thank you for your reply.

Since changes can not be made to the old documentation will will keep having 2 indexes. This comment closes #1140

If you need anything, feel free to reach out. Have a great EoY and an amazing 20/20
